### PR TITLE
Remove deprecated nGram, edgeNGram, htmlStrip Tokenfilters names in favour for ngram, edge_ngram, and html_strip

### PR DESCRIPTION
**Pull request by mkleen**
## Checklist

 - [x] User relevant changes are recorded in ``CHANGES.txt``
 - [x] Touched code is covered by tests
 - [x] Documentation has been updated if necessary
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

**Comment by mkleen** [comment]
retest this please

**Comment by mkleen** [comment]
It depends on the lucene 8 changes. I will squash it together as soon as lucene 8 makes it into master.

File: blackbox/docs/appendices/release-notes/unreleased.rst [hunk 1]
```diff
@@ -41,6 +41,8 @@ Unreleased Changes
 Breaking Changes
 ================
 
+- Remove deprecated nGram, edgeNGram, htmlStrip token filter names in favour for ngram, edge_ngram, and html_strip
```
Comment (seut) [inline blackbox/docs/appendices/release-notes/unreleased.rst]: Please use past tense, surround nGram etc with double backticks, respect the 79 char limit per line and add a `.` and the end of a sentence.

Comment (seut) [reply blackbox/docs/appendices/release-notes/unreleased.rst]: `html_strip` is a char filter.

File: blackbox/docs/appendices/release-notes/unreleased.rst [hunk 2]
```diff
@@ -41,6 +41,9 @@ Unreleased Changes
 Breaking Changes
 ================
 
+- Removed deprecated nGram, edgeNGram, htmlStrip filter names in favour
+  for ngram, edge_ngram, and html_strip.
```
Comment (seut) [inline blackbox/docs/appendices/release-notes/unreleased.rst]:
```suggestion
Removed deprecated CamelCase ``nGram``, ``edgeNGram`` token filter and ``htmlStrip`` char filter, they are superseded by ``ngram``, ``edge_ngram`` and ``html_strip``.
```

File: blackbox/docs/appendices/release-notes/unreleased.rst [hunk 3]
```diff
@@ -41,6 +41,11 @@ Unreleased Changes
 Breaking Changes
 ================
 
+- Removed deprecated ``nGram``, ``edgeNGram`` token filter and ``htmlStrip``
+  char filter, they are superseded by ``ngram``, ``edge_ngram``and
```
Comment (seut) [inline blackbox/docs/appendices/release-notes/unreleased.rst]: missing space before *and*

Comment (seut) [reply blackbox/docs/appendices/release-notes/unreleased.rst]: Please move to `Deprecations`, these changes are not breaking.

**Comment by mkleen** [comment]
retest this please

File: blackbox/docs/appendices/release-notes/unreleased.rst [hunk 4]
```diff
@@ -122,6 +123,10 @@ Breaking Changes
 Deprecations
 ============
 
+- Removed deprecated ``nGram``, ``edgeNGram`` token filter and ``htmlStrip``
```
Comment (seut) [inline blackbox/docs/appendices/release-notes/unreleased.rst]: Sorry but please move to ``Changes`` as we do not deprecate but remove them now.

File: sql/src/test/java/io/crate/integrationtests/InformationSchemaTest.java [hunk 5]
```diff
@@ -45,6 +46,7 @@
 import static org.hamcrest.Matchers.arrayContaining;
 import static org.hamcrest.Matchers.is;
 
+@UseJdbc(0)
```
Comment (seut) [inline sql/src/test/java/io/crate/integrationtests/InformationSchemaTest.java]: added by intend?

File: blackbox/docs/appendices/release-notes/unreleased.rst [hunk 6]
```diff
@@ -134,6 +135,10 @@ Changes
 - Added :ref:`current_setting <scalar_current_setting>` system information
   scalar function that yields the current value of the setting.
 
+- Removed deprecated ``nGram``, ``edgeNGram`` token filter and ``htmlStrip``
```
Comment (seut) [inline blackbox/docs/appendices/release-notes/unreleased.rst]: we always put new entries on top (the related commit will also be on top ;)

**Comment by mkleen** [comment]
retest this please

**Pull request merged**